xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -763,7 +763,7 @@ std::string CGUIWindowVideoBase::GetResumeString(const CFileItem &item)
 
 bool CGUIWindowVideoBase::ShowResumeMenu(CFileItem &item)
 {
-  if (!item.m_bIsFolder && !item.IsPVR())
+  if (!item.m_bIsFolder && !item.IsPVR() || item.IsPlugin())
```

Comment:
m_bIsFolder depends on the value specified for isFolder at  xbmcplugin.addDirectoryItem (https://codedocs.xyz/xbmc/xbmc/group__python__xbmcplugin.html#ga4c357cf7d3dbd4e2823755ecfd7dfa01)